joda-time/src/main/java/org/joda/time/MutableInterval.java

```diff
@@ -294,7 +294,7 @@ public void setStart(ReadableInstant start) {
      * @throws IllegalArgumentException if the end is before the start
      */
         public void setEndMillis(long endInstant) {
```

Comment:
Indentation fix needed for consistent formatting

Reply:
@mariam606 please fix this case as well as all others.